.eslintrc.js

```diff
@@ -10,4 +10,7 @@ module.exports = {
     sourceType: "module",
   },
   plugins: ["@typescript-eslint"],
+  rules: {
+    "no-unused-vars": "off",
```

Comment:
@marcelveldt My suggestion is to keep this on and add the ignore lines where needed. This rule helps